docs/schema/aws.md

```diff
@@ -1570,6 +1583,35 @@ Representation of a generic Network Interface.  Currently however, we only creat
         (NetworkInterface)-[TAGGED]->(AWSTag)
         ```
 
+## PeeringConnection
+
+Representation of an AWS [PeeringConnection](https://docs.aws.amazon.com/vpc/latest/peering/what-is-vpc-peering.html)
```

Comment:
Just double-checking, but a PeeringConnection has no associated AWS API data type?

If this is the case, maybe add in a comment saying that a PeeringConnection node is an invented cartography construct that we use to more easily create route tables.

Also consider calling it an AWSPeeringConnection in case any of the other public cloud products have a similar concept.

Reply:
The acutual object is VpcPeeringConnection, but I thought it was superfluous to add the VPC prefix -- I was already fighting a lot with line lengths, every chatacter counts.

Reply:
AWSPeeringConnection added in 97c7187